semantic-python/src/Language/Python/Core.hs

```diff
@@ -300,17 +300,16 @@ instance Compile Py.NotOperator
 instance Compile Py.ParenthesizedExpression
 
 instance Compile Py.PassStatement where
-  compile it@Py.PassStatement {} _ = locate it $ Core.unit
+  compile it@Py.PassStatement {} cc _ = locate it Core.unit >>= cc
```

Comment:
Confirming my understanding, it looks like what we’re doing here is returning (via CPS) the `unit` while ignoring the following statements, implementing `pass`’ semantics as something which breaks/escapes control flow locally.

That is really cool 👍